Tabs support

artemsen opened the pull request:
@dmitry-j-mikhin, please take a look.

artemsen committed:
Use 64-bit unsigned long for window id

Signed-off-by: Artem Senichev <artemsen@gmail.com>

dmitry-j-mikhin commented:
> @dmitry-j-mikhin, please take a look.

Looks good and consistent. And one more thing, just to discuss. wnd_id += cnt_id can be wrapped around, and there is a possibility of matching with another window/tab, although the probability of this is very low. You can limit the hash range to reduce this likelihood, but it may not be worth it)

artemsen committed:
Add per-tab mode for web browsers

Allows to store keyboard layout for each tab separately.
Currently only Firefox and Chromium are supported.
Based on Dmitry Mikhin's implementation (PR #6).

Co-developed-by: Dmitry Mikhin <dmikhin@webmonitorx.ru>
Signed-off-by: Artem Senichev <artemsen@gmail.com>
